website_product_filters/controllers/main.py

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# pylint: disable=expression-not-assigned
```

Comment:
Because some there are some `list.append(stuff)` sentences that given a condition they are called, but there is no assignment of the result.
See example : https://github.com/Vauxoo/ecommerce/blob/8.0/product_filters/models/product.py#L151